### ISSUE TITLE
[DOC] Issue with databricks_workspace_binding argument securable_type

### ISSUE THREAD

**Issue by Lorenzvanijperen** [databricks/terraform-provider-databricks#4307]
### Affected Resource(s)
https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/workspace_binding#securable_type-1

### Expected Details
Docs shows:

[securable_type](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/workspace_binding#securable_type-1) - Type of securable. Can be catalog, external-location or storage-credential. Default to catalog. Change forces creation of a new resource.

Code shows: 
Error: expected securable_type to be one of ["catalog" "external_location" "storage_credential"], got external-location

### List of things to potentially add/remove

This is a list of things to manipulate in the docs:

- storage-credential --> storage_credential
- external-location --> external_location

**Comment by alexott** [comment]
addressed in https://github.com/databricks/terraform-provider-databricks/pull/4293

**Comment by Farhan047** [comment]
> addressed in #4293

I believe this should also resolve the below error :
```
 Error: expected securable_type to be one of ["catalog" "external_location" "storage_credential"], got credential
```
[The API supports it](https://docs.databricks.com/api/azure/workspace/workspacebindings/updatebindings#securable_type), but the current version of the provider doesn't, this PR #4293 should help address that.

FYI, I am trying to create a service credential with a workspace binding, for a Managed Identity (`azurerm_databricks_access_connector`) as a Secrets User on an Azure Key Vault.

Question, do we know when you are planning to merge this PR ? And a timeline on the release this change will be part of ?

**Comment by alexott** [comment]
Will be resolved by merging #4328 

@Farhan047 release with the fix will be tomorrow (most probably)

**Comment by Farhan047** [comment]
> Will be resolved by merging #4328
> 
> @Farhan047 release with the fix will be tomorrow (most probably)

Thanks @alexott
